rllib/agents/marwil/marwil_tf_policy.py

```diff
@@ -67,7 +67,8 @@ def postprocess_advantages(policy,
         # input_dict.
         # Create an input dict according to the Model's requirements.
         index = "last" if SampleBatch.NEXT_OBS in sample_batch.data else -1
-        input_dict = policy.model.get_input_dict(sample_batch, index=index)
+        input_dict = sample_batch.get_single_step_input_dict(
```

Comment:
Moved this helper method to SampleBatch (from ModelV2), where it's more fitting.